packages/flutter/lib/src/cupertino/magnifier.dart

```diff
@@ -0,0 +1,308 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+import 'package:flutter/widgets.dart';
+
+/// A [CupertinoMagnifier], specifically for integrating with [SelectionOverlay].
+///
+/// Delegates styling to [CupertinoMagnifier] with its position depending on
+/// [magnifierOverlayInfoBearer].
+///
+/// Specifically, the [CupertinoTextMagnifier] follows the following rules.
+/// [CupertinoTextMagnifier]:
+/// - is positioned horizontally outside the screen width, with _kHorizontalScreenEdgePadding padding.
```

Comment:
Nit: Capitalization

Reply:
re: this and the comment above. Justin and I tried to format these so they read like setences, all prefixed by the CupertinoTextMagnifier and then some action statement